postgresql-demo-apb/apb.yml

```diff
@@ -0,0 +1,22 @@
+id: 26355d1a-0301-4841-85af-536a4d3afaa9
+name: ansibleplaybookbundle/postgresql-demo-apb
```

Comment:
name needs to be split:
`
name: postgresql-demo-apb
image: ansibleplaybookbundle/postgresql-demo-apb
`
You also need at least:
`
tags:
 - database
`